server/auth/types/saml/saml_auth.ts

```diff
@@ -104,7 +110,7 @@ export class SamlAuthentication extends AuthenticationType {
       cookie.authType === AuthType.SAML &&
       cookie.username &&
       cookie.expiryTime &&
-      cookie.credentials?.authHeaderValue
+      (cookie.credentials?.authHeaderValue || cookie.credentials?.authHeaderValueCompressed)
```

Comment:
What would you think about inserting a cookie splitter that could provide head room of these requests?

Reply:
Would be an option, but we cannot do this inside this code. From this code, we can only write exactly one cookie.